support/ebpf/interpreter_dispatcher.ebpf.c

```diff
@@ -124,6 +126,71 @@ bpf_map_def SEC("maps") apm_int_procs = {
   .max_entries = 128,
 };
 
+bpf_map_def SEC("maps") go_procs = {
+  .type        = BPF_MAP_TYPE_HASH,
+  .key_size    = sizeof(pid_t),
+  .value_size  = sizeof(GoCustomLabelsOffsets),
+  .max_entries = 128,
+};
+
+static inline __attribute__((__always_inline__)) void *
+get_m_ptr(struct GoCustomLabelsOffsets *offs, UnwindState *state)
+{
+  long res;
+
+  size_t g_addr;
+#if defined(__x86_64__)
+  u64 g_addr_offset = 0xfffffffffffffff8;
+  void *tls_base    = NULL;
+  res               = tsd_get_base(&tls_base);
+  if (res < 0) {
+    DEBUG_PRINT("cl: failed to get tsd base; can't read m_ptr");
+    return NULL;
+  }
+
+  res = bpf_probe_read_user(&g_addr, sizeof(void *), (void *)((u64)tls_base + g_addr_offset));
```

Comment:
```suggestion
  res = bpf_probe_read_user(&g_addr, sizeof(void *), (void *)((u64)tls_base - 8));
```
Or just this if the Go ABI is fixed and unlikely to change?